WORKSPACE

```diff
@@ -715,6 +715,17 @@ http_archive(
     ],
 )
 
+# This must be kept in sync with src/main/java/com/google/devtools/build/lib/bazel/rules/java/jdk.WORKSPACE.
+http_archive(
+    name = "remote_java_tools_linux_beta_for_testing",
+    patch_cmds = EXPORT_WORKSPACE_IN_BUILD_FILE,
+    patch_cmds_win = EXPORT_WORKSPACE_IN_BUILD_FILE_WIN,
+    sha256 = "74d30ccf161c58bb69db9b2171c954a0563b2d1ff6f5831debbe71ced105c388",
+    urls = [
+        "https://github.com/davido/java_tools/releases/download/javac11-v11.0/java_tools_javac11_linux-v11.0.zip",
+    ],
+)
+
```

Comment:
Doesn't this mean, that before every release, we have to change this archive? Because of the version, i.e. rc1 and because of the sha sum?

Reply:
> Doesn't this mean, that before every release, we have to change this archive?

Yes, that's correct.